tmgmt_zanata.ui.inc

```diff
@@ -65,6 +65,13 @@ class TMGMTZanataTranslatorUIController extends TMGMTDefaultTranslatorUIControll
       '#maxlength' => 40,
       '#default_value' => $translator->getSetting('version'),
     );
+    $form['transcheck'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Translation Periodic Check'),
+      '#description' => t(
+        'Pull translations delta automatically.'),
```

Comment:
Since there may be a non-technical person trying to configure the server, I suggest `'Automatically fetch new translations'`, and include the frequency of the check.